src/main/webapp/search-interview.js

```diff
@@ -54,48 +66,76 @@ function selectInterview(interviewer) {
   }
   let utcStartTime = interviewer.getAttribute('data-utc');
   let position = document.getElementById('position').value;
-  if (confirm(
-      `You selected: ${date} from ${time} with a ` +
-      `${company} ${job}. ` +
-      `Click OK if you wish to proceed.`)) {
-    alert(
-      `You have scheduled a ${position} interview on ${date}` +
-      ` from ${time} with a ${company} ` +
-      `${job}. Check your email for more ` +
-      `information.`);
-    if (company === '<Not specified>') {
-      company = '';
+  let role = selectedRole();
+  if (role === 'Interviewee') {
+    if (confirm(
+        `You selected: ${date} from ${time} with a ` +
+        `${company} ${job}. ` +
+        `Click OK if you wish to proceed.`)) {
+      alert(
+        `You have scheduled a ${position} interview on ${date}` +
+        ` from ${time} with a ${company} ` +
+        `${job}. Check your email for more ` +
+        `information.`);
+      updateDatastore('POST', company, job, utcStartTime);
     }
-    if (job === '<Not specified>') {
-      job = '';
+  }
+  if (role === 'Shadow') {
+    if (confirm(
+        `You selected: ${date} from ${time} with a ` +
+        `${company} ${job}. ` +
+        `Click OK if you wish to proceed.`)) {
+      alert(
+        `You will shadow a ${position} interview on ${date}` +
+        ` from ${time} with a ${company} ` +
+        `${job}. Check your email for more ` +
+        `information.`);
+      updateDatastore('PUT', company, job, utcStartTime);
     }
-    let requestObject = {
-      company: company,
-      job: job,
-      utcStartTime: utcStartTime,
-      position: selectedPosition()
-    };
-    let requestBody = JSON.stringify(requestObject);
-    let request = new Request('/scheduled-interviews', {method: 'POST', body: requestBody});
-    fetch(request).then(() => {window.location.replace('/scheduled-interviews.html');});
   }
 }
 
+function updateDatastore(method, company, job, utcStartTime) {
+  if (company === '<Not specified>') {
+    company = '';
```

Comment:
Having multiple representations of "not specified" is a pretty bad code smell, but I know we're not going to have time to fix this.

Reply:
If it were to be fixed, we'd put <Not specified> everywhere instead of an empty string and then we'd have to deal with the HTML processing it as a tag again.